hoomd/CMakeLists.txt

```diff
@@ -153,6 +154,14 @@ set(_hoomd_headers
     Variant.h
     VectorMath.h
     WarpTools.cuh
+    filter/ParticleFilter.h
+    filter/ParticleFilterAll.h
```

Comment:
This will not install these headers in the proper location. The later cmake command `install(FILES ${_hoomd_headers} ...` installs all files into the same directory regardless of their absolute path, so there is no `filter` subdirectory in the installed header location.

We either need to put all these headers in the `hoomd/` directory or add a new `install` commands to install these headers in the proper location and remove them from the `_hoomd_headers` list.

Reply:
I have attempted to resolve this. i am not familiar with CMake so if this is not correct, let me know.

Reply:
You were just missing the `add_subdirectory(filter)`. I pushed a commit to the branch that does this.

Reply:
Ah that is a much simpler fix. I am sorry for making you revert my changes and perform the appropriate ones.

Reply:
I didn't revert any changes. The `CMakeLists.txt` file you added to `filters` is what `add_subdirectory` calls.